lib/game.rb

```diff
@@ -0,0 +1,16 @@
+class Game
+  attr_reader :name,
+              :cost,
+              :national_drawing
```

Comment:
Because we have defined the method `national_drawing?` below, we do not need this attr_reader